src/services/garbageCleaningService.js

```diff
@@ -0,0 +1,63 @@
+import api from "@api/api";
+import encryptionService from "./encryptionService";
+import indexedDB from "@store/indexedDB";
+import localforage from "localforage";
+import store from "@store/store";
+import { clearMessageIdsToLocalLimit } from "@store/values/Conversations";
+import { clearMessagesToLocalLimit } from "@store/values/Messages";
+import { setSelectedConversation } from "@store/values/SelectedConversation";
+import { setUserIsLoggedIn } from "@store/values/UserIsLoggedIn";
+import { updateNetworkState } from "@store/values/NetworkState";
+
+class GarbageCleaningService {
+  async #clearAccountData() {
+    await encryptionService.clearStoredAccount();
+  }
+
+  #clearAccountDataSync() {
+    encryptionService.clearStoredAccount();
+  }
+
+  #clearLocalMessageIds(cid) {
+    store.dispatch(clearMessageIdsToLocalLimit(cid));
+  }
+
+  #clearLocalMessages(cid) {
+    store.dispatch(clearMessagesToLocalLimit(cid));
+  }
+
+  async clearConversationMessages(cid) {
```

Comment:
do we need separate methods for 
`#clearLocalMessageIds` and `#clearLocalMessageIds` ? Will it be a case where we need to call it separately? Probably not.

so we can optimise

```
async clearConversationMessages(cid) {
  if (!cid) return;
  store.dispatch(clearMessagesToLocalLimit(cid));
  store.dispatch(clearMessageIdsToLocalLimit(cid));
}
```